### ISSUE TITLE
Running `apt-get update` hangs on `armv7-unknown-linux-gnueabihf` target

### ISSUE THREAD

**Issue by alloncm** [cross-rs/cross#1320]
### Checklist

- [X] I've looked through the [issues and pull requests](https://github.com/cross-rs/cross/issues?q=) for similar reports

### Describe your issue

Cross started to just hang for really long time (more than 3 hours until I closed it on my windows laptop) when running `apt-get update` in the pre-build script on `armv7-unknown-linux-gnueabihf` target.

### What target(s) are you cross-compiling for?

armv7-unknown-linux-gnueabihf

### Which operating system is the host (e.g computer cross is on) running?

- [ ] macOS
- [X] Windows
- [ ] Linux / BSD
- [ ] other OS (specify in description)

### What architecture is the host?

- [X] x86_64 / AMD64
- [ ] arm32
- [ ] arm64 (including Mac M1)

### What container engine is cross using?

- [X] docker
- [ ] podman
- [ ] other container engine (specify in description)

### cross version

cross 0.2.5

### Example

```sh
docker run -it ghcr.io/cross-rs/armv7-unknown-linux-gnueabihf:0.2.5 /bin/bash
```
and inside the container 
```sh
apt-get update
```

### Additional information / notes

This issue started in the last 2-3 weeks so I guess it is something with the base image.
When updating to `ghcr.io/cross-rs/armv7-unknown-linux-gnueabihf:main` (the main tag) it works fine, so I checked the diff between 0.2.5 and main and saw that the base image is newer by 4 years (ubuntu:16.04 vs ubuntu:20.04).

**Comment by Emilgardis** [comment]
is it the same on other targets? This works for me on windows (except some fails due to 16.04 not being signed correctly anymore)


can you try

`docker run -it ubuntu:16.04`
and
`apt-get update`
there?

If that fails, try `docker run --network host -it ubuntu:16.04`, if that does solve it, you can add it to the command cross runs with `$env:CROSS_CONTAINER_OPTS="--network host"`

**Comment by alloncm** [comment]
Thanks for the quick replay!

Just verified and the issue happens also with `aarch64-unknown-linux-gnu:0.2.5` as target (image - ghcr.io/cross-rs/aarch64-unknown-linux-gnu:0.2.5).

I also tried your suggestions and ubuntu:16.04 with or without `--network host` still fails to `apt-get update` is reasonable time.

**Comment by Emilgardis** [comment]
alright, there's one other thing you can try, `apt-get -o Acquire::ForceIPv4=true -o Debug::Acquire::http=true update`

also, are you using the latest version of docker for windows?


when it gets stuck, where does it hang? Is there any output? How about with the above command?

**Comment by alloncm** [comment]
This is the output of the command:
<details>

```sh
root@docker-desktop:/# apt-get -o Acquire::ForceIPv4=true -o Debug::Acquire::http=true update
0% [Connecting to archive.ubuntu.com] [Connecting to security.ubuntu.com]GET /ubuntu/dists/xenial/InRelease HTTP/1.1
Host: archive.ubuntu.com
Cache-Control: max-age=0
Accept: text/*
User-Agent: Debian APT-HTTP/1.3 (1.2.35)


GET /ubuntu/dists/xenial-security/InRelease HTTP/1.1
Host: security.ubuntu.com
Cache-Control: max-age=0
Accept: text/*
User-Agent: Debian APT-HTTP/1.3 (1.2.35)


0% [Waiting for headers] [Waiting for headers]Answer for: http://archive.ubuntu.com/ubuntu/dists/xenial/InRelease
HTTP/1.1 200 OK
Date: Sat, 26 Aug 2023 20:45:37 GMT
Server: Apache/2.4.29 (Ubuntu)
Last-Modified: Thu, 21 Apr 2016 23:24:48 GMT
ETag: "3c43e-5310704319fe1"
Accept-Ranges: bytes
Content-Length: 246846
Cache-Control: max-age=0, proxy-revalidate
Expires: Sat, 26 Aug 2023 20:45:37 GMT
Connection: close

Get:1 http://archive.ubuntu.com/ubuntu xenial InRelease [247 kB]
0% [1 InRelease 2608 B/247 kB 1%] [Waiting for headers]Answer for: http://security.ubuntu.com/ubuntu/dists/xenial-security/InRelease
HTTP/1.1 200 OK
Date: Sat, 26 Aug 2023 20:45:37 GMT
Server: Apache/2.4.29 (Ubuntu)
Last-Modified: Sat, 26 Aug 2023 20:03:00 GMT
ETag: "185bd-603d8f2b48500"
Accept-Ranges: bytes
Content-Length: 99773
Cache-Control: max-age=742, s-maxage=3300, proxy-revalidate
Expires: Sat, 26 Aug 2023 20:58:00 GMT
Connection: close

Get:2 http://security.ubuntu.com/ubuntu xenial-security InRelease [99.8 kB]
Ign:2 http://security.ubuntu.com/ubuntu xenial-security InRelease
0% [1 InRelease 247 kB/247 kB 100%]GET /ubuntu/dists/xenial-security/Release HTTP/1.1
Host: security.ubuntu.com
Cache-Control: max-age=0
Accept: text/*
User-Agent: Debian APT-HTTP/1.3 (1.2.35)


GET /ubuntu/dists/xenial-security/Release HTTP/1.1
Host: security.ubuntu.com
Cache-Control: max-age=0
Accept: text/*
User-Agent: Debian APT-HTTP/1.3 (1.2.35)


Ign:1 http://archive.ubuntu.com/ubuntu xenial InRelease
0% [Connecting to archive.ubuntu.com (185.125.190.36)] [Waiting for headers]GET /ubuntu/dists/xenial-updates/InRelease HTTP/1.1
Host: archive.ubuntu.com
Cache-Control: max-age=0
Accept: text/*
User-Agent: Debian APT-HTTP/1.3 (1.2.35)


Answer for: http://security.ubuntu.com/ubuntu/dists/xenial-security/Release
HTTP/1.1 200 OK
Date: Sat, 26 Aug 2023 20:47:39 GMT
Server: Apache/2.4.52 (Ubuntu)
Last-Modified: Sat, 26 Aug 2023 20:03:00 GMT
ETag: "181f8-603d8f2b48500"
Accept-Ranges: bytes
Content-Length: 98808
Cache-Control: max-age=620, s-maxage=3300, proxy-revalidate
Expires: Sat, 26 Aug 2023 20:58:00 GMT
Connection: close

Get:3 http://security.ubuntu.com/ubuntu xenial-security Release [98.8 kB]
0% [Waiting for headers] [3 Release 1132 B/98.8 kB 1%]Answer for: http://archive.ubuntu.com/ubuntu/dists/xenial-updates/InRelease
HTTP/1.1 200 OK
Date: Sat, 26 Aug 2023 20:47:39 GMT
Server: Apache/2.4.52 (Ubuntu)
Last-Modified: Sat, 26 Aug 2023 20:03:00 GMT
ETag: "185bb-603d8f2b48500"
Accept-Ranges: bytes
Content-Length: 99771
Cache-Control: max-age=0, proxy-revalidate, s-maxage=3300
Expires: Sat, 26 Aug 2023 20:47:39 GMT
Connection: close

Get:4 http://archive.ubuntu.com/ubuntu xenial-updates InRelease [99.8 kB]
Ign:3 http://security.ubuntu.com/ubuntu xenial-security Release
0% [4 InRelease 99.8 kB/99.8 kB 100%]GET /ubuntu/dists/xenial-security/main/binary-amd64/Packages.gz HTTP/1.1
Host: security.ubuntu.com
Cache-Control: max-age=0
User-Agent: Debian APT-HTTP/1.3 (1.2.35)


GET /ubuntu/dists/xenial-security/main/binary-amd64/Packages.gz HTTP/1.1
Host: security.ubuntu.com
Cache-Control: max-age=0
User-Agent: Debian APT-HTTP/1.3 (1.2.35)


Ign:4 http://archive.ubuntu.com/ubuntu xenial-updates InRelease
0% [Waiting for headers]GET /ubuntu/dists/xenial-backports/InRelease HTTP/1.1
Host: archive.ubuntu.com
Cache-Control: max-age=0
Accept: text/*
User-Agent: Debian APT-HTTP/1.3 (1.2.35)


Answer for: http://security.ubuntu.com/ubuntu/dists/xenial-security/main/binary-amd64/Packages.gz
HTTP/1.1 200 OK
Date: Sat, 26 Aug 2023 20:49:40 GMT
Server: Apache/2.4.52 (Ubuntu)
Last-Modified: Sat, 26 Aug 2023 20:03:00 GMT
ETag: "1f5076-603d8f2b48500"
Accept-Ranges: bytes
Content-Length: 2052214
Cache-Control: max-age=499, s-maxage=3300, proxy-revalidate
Expires: Sat, 26 Aug 2023 20:58:00 GMT
Connection: close
Content-Type: application/x-gzip

Get:5 http://security.ubuntu.com/ubuntu xenial-security/main amd64 Packages [2052 kB]
0% [Waiting for headers] [5 Packages 3715 B/2052 kB 0%]Answer for: http://archive.ubuntu.com/ubuntu/dists/xenial-backports/InRelease
HTTP/1.1 200 OK
Date: Sat, 26 Aug 2023 20:49:40 GMT
Server: Apache/2.4.29 (Ubuntu)
Last-Modified: Sat, 26 Aug 2023 20:03:00 GMT
ETag: "17c77-603d8f2b48500"
Accept-Ranges: bytes
Content-Length: 97399
Cache-Control: max-age=0, proxy-revalidate
Expires: Sat, 26 Aug 2023 20:49:40 GMT
Connection: close

Get:6 http://archive.ubuntu.com/ubuntu xenial-backports InRelease [97.4 kB]
Ign:6 http://archive.ubuntu.com/ubuntu xenial-backports InRelease
0% [5 Packages 2052 kB/2052 kB 100%]GET /ubuntu/dists/xenial/Release HTTP/1.1
Host: archive.ubuntu.com
Cache-Control: max-age=0
Accept: text/*
User-Agent: Debian APT-HTTP/1.3 (1.2.35)


Answer for: http://archive.ubuntu.com/ubuntu/dists/xenial/Release
HTTP/1.1 200 OK
Date: Sat, 26 Aug 2023 20:51:41 GMT
Server: Apache/2.4.52 (Ubuntu)
Last-Modified: Thu, 21 Apr 2016 23:24:48 GMT
ETag: "3c068-5310704319fe1"
Accept-Ranges: bytes
Content-Length: 245864
Cache-Control: max-age=0, proxy-revalidate, s-maxage=3300
Expires: Sat, 26 Aug 2023 20:51:41 GMT
Connection: close

Get:7 http://archive.ubuntu.com/ubuntu xenial Release [246 kB]
Ign:5 http://security.ubuntu.com/ubuntu xenial-security/main amd64 Packages
0% [7 Release 160 kB/246 kB 65%]GET /ubuntu/dists/xenial-security/main/binary-all/Packages.gz HTTP/1.1
Host: security.ubuntu.com
Cache-Control: max-age=0
User-Agent: Debian APT-HTTP/1.3 (1.2.35)


GET /ubuntu/dists/xenial-security/main/binary-all/Packages.gz HTTP/1.1
Host: security.ubuntu.com
Cache-Control: max-age=0
User-Agent: Debian APT-HTTP/1.3 (1.2.35)


Answer for: http://security.ubuntu.com/ubuntu/dists/xenial-security/main/binary-all/Packages.gz
HTTP/1.1 404 Not Found
Date: Sat, 26 Aug 2023 20:51:42 GMT
Server: Apache/2.4.52 (Ubuntu)
Content-Length: 281
Connection: close
Content-Type: text/html; charset=iso-8859-1

Ign:8 http://security.ubuntu.com/ubuntu xenial-security/main all Packages
0% [7 Release 246 kB/246 kB 100%]GET /ubuntu/dists/xenial-security/restricted/binary-amd64/Packages.gz HTTP/1.1
Host: security.ubuntu.com
Cache-Control: max-age=0
User-Agent: Debian APT-HTTP/1.3 (1.2.35)


Ign:7 http://archive.ubuntu.com/ubuntu xenial Release
0% [Connecting to archive.ubuntu.com (185.125.190.36)]GET /ubuntu/dists/xenial-updates/Release HTTP/1.1
Host: archive.ubuntu.com
Cache-Control: max-age=0
Accept: text/*
User-Agent: Debian APT-HTTP/1.3 (1.2.35)


Ign:9 http://security.ubuntu.com/ubuntu xenial-security/restricted amd64 Packages
0% [Waiting for headers]GET /ubuntu/dists/xenial-security/restricted/binary-all/Packages.gz HTTP/1.1
Host: security.ubuntu.com
Cache-Control: max-age=0
User-Agent: Debian APT-HTTP/1.3 (1.2.35)


Answer for: http://archive.ubuntu.com/ubuntu/dists/xenial-updates/Release
HTTP/1.1 200 OK
Date: Sat, 26 Aug 2023 20:53:42 GMT
Server: Apache/2.4.52 (Ubuntu)
Last-Modified: Sat, 26 Aug 2023 20:03:00 GMT
ETag: "181f6-603d8f2b48500"
Accept-Ranges: bytes
Content-Length: 98806
Cache-Control: max-age=0, proxy-revalidate, s-maxage=3300
Expires: Sat, 26 Aug 2023 20:53:42 GMT
Connection: close

Get:10 http://archive.ubuntu.com/ubuntu xenial-updates Release [98.8 kB]
0% [10 Release 2394 B/98.8 kB 2%] [Waiting for headers]Answer for: http://security.ubuntu.com/ubuntu/dists/xenial-security/restricted/binary-all/Packages.gz
HTTP/1.1 404 Not Found
Date: Sat, 26 Aug 2023 20:53:42 GMT
Server: Apache/2.4.29 (Ubuntu)
Content-Length: 281
Connection: close
Content-Type: text/html; charset=iso-8859-1

Ign:11 http://security.ubuntu.com/ubuntu xenial-security/restricted all Packages
0% [10 Release 29.6 kB/98.8 kB 30%]GET /ubuntu/dists/xenial-security/universe/binary-amd64/Packages.gz HTTP/1.1
Host: security.ubuntu.com
Cache-Control: max-age=0
User-Agent: Debian APT-HTTP/1.3 (1.2.35)


Ign:12 http://security.ubuntu.com/ubuntu xenial-security/universe amd64 Packages
0% [10 Release 98.8 kB/98.8 kB 100%] [Connecting to security.ubuntu.com (91.189.91.39)]GET /ubuntu/dists/xenial-security/universe/binary-all/Packages.gz HTTP/1.1
Host: security.ubuntu.com
Cache-Control: max-age=0
User-Agent: Debian APT-HTTP/1.3 (1.2.35)


Ign:10 http://archive.ubuntu.com/ubuntu xenial-updates Release
0% [Waiting for headers]GET /ubuntu/dists/xenial-backports/Release HTTP/1.1
Host: archive.ubuntu.com
Cache-Control: max-age=0
Accept: text/*
User-Agent: Debian APT-HTTP/1.3 (1.2.35)


Answer for: http://security.ubuntu.com/ubuntu/dists/xenial-security/universe/binary-all/Packages.gz
HTTP/1.1 404 Not Found
Date: Sat, 26 Aug 2023 20:55:43 GMT
Server: Apache/2.4.52 (Ubuntu)
Content-Length: 281
Connection: close
Content-Type: text/html; charset=iso-8859-1

Ign:13 http://security.ubuntu.com/ubuntu xenial-security/universe all Packages
0% [Waiting for headers]GET /ubuntu/dists/xenial-security/multiverse/binary-amd64/Packages.gz HTTP/1.1
Host: security.ubuntu.com
Cache-Control: max-age=0
User-Agent: Debian APT-HTTP/1.3 (1.2.35)


Answer for: http://archive.ubuntu.com/ubuntu/dists/xenial-backports/Release
HTTP/1.1 200 OK
Date: Sat, 26 Aug 2023 20:55:43 GMT
Server: Apache/2.4.52 (Ubuntu)
Last-Modified: Sat, 26 Aug 2023 20:03:00 GMT
ETag: "178b2-603d8f2b48500"
Accept-Ranges: bytes
Content-Length: 96434
Cache-Control: max-age=0, proxy-revalidate, s-maxage=3300
Expires: Sat, 26 Aug 2023 20:55:43 GMT
Connection: close

Get:14 http://archive.ubuntu.com/ubuntu xenial-backports Release [96.4 kB]
Ign:15 http://security.ubuntu.com/ubuntu xenial-security/multiverse amd64 Packages
0% [14 Release 96.4 kB/96.4 kB 100%]GET /ubuntu/dists/xenial-security/multiverse/binary-all/Packages.gz HTTP/1.1
Host: security.ubuntu.com
Cache-Control: max-age=0
User-Agent: Debian APT-HTTP/1.3 (1.2.35)


Answer for: http://security.ubuntu.com/ubuntu/dists/xenial-security/multiverse/binary-all/Packages.gz
HTTP/1.1 404 Not Found
Date: Sat, 26 Aug 2023 20:57:43 GMT
Server: Apache/2.4.29 (Ubuntu)
Content-Length: 281
Connection: close
Content-Type: text/html; charset=iso-8859-1

Ign:16 http://security.ubuntu.com/ubuntu xenial-security/multiverse all Packages
0% [14 Release 96.4 kB/96.4 kB 100%]GET /ubuntu/dists/xenial-security/main/binary-amd64/Packages.xz HTTP/1.1
Host: security.ubuntu.com
Cache-Control: max-age=0
User-Agent: Debian APT-HTTP/1.3 (1.2.35)


Ign:14 http://archive.ubuntu.com/ubuntu xenial-backports Release
4% [Connecting to archive.ubuntu.com (185.125.190.36)]GET /ubuntu/dists/xenial/main/binary-amd64/Packages.gz HTTP/1.1
Host: archive.ubuntu.com
Cache-Control: max-age=0
User-Agent: Debian APT-HTTP/1.3 (1.2.35)


Answer for: http://archive.ubuntu.com/ubuntu/dists/xenial/main/binary-amd64/Packages.gz
HTTP/1.1 200 OK
Date: Sat, 26 Aug 2023 20:57:44 GMT
Server: Apache/2.4.52 (Ubuntu)
Last-Modified: Thu, 21 Apr 2016 23:24:48 GMT
ETag: "17c5e1-5310704319fe1"
Accept-Ranges: bytes
Content-Length: 1557985
Cache-Control: max-age=0, proxy-revalidate, s-maxage=3300
Expires: Sat, 26 Aug 2023 20:57:44 GMT
Connection: close
Content-Type: application/x-gzip

Get:17 http://archive.ubuntu.com/ubuntu xenial/main amd64 Packages [1558 kB]
Ign:5 http://security.ubuntu.com/ubuntu xenial-security/main amd64 Packages
51% [17 Packages 1558 kB/1558 kB 100%]GET /ubuntu/dists/xenial-security/main/binary-all/Packages.xz HTTP/1.1
Host: security.ubuntu.com
Cache-Control: max-age=0
User-Agent: Debian APT-HTTP/1.3 (1.2.35)


Answer for: http://security.ubuntu.com/ubuntu/dists/xenial-security/main/binary-all/Packages.xz
HTTP/1.1 404 Not Found
Date: Sat, 26 Aug 2023 20:59:44 GMT
Server: Apache/2.4.29 (Ubuntu)
Content-Length: 281
Connection: close
Content-Type: text/html; charset=iso-8859-1

Ign:8 http://security.ubuntu.com/ubuntu xenial-security/main all Packages
51% [17 Packages 1558 kB/1558 kB 100%]GET /ubuntu/dists/xenial-security/restricted/binary-amd64/Packages.xz HTTP/1.1
Host: security.ubuntu.com
Cache-Control: max-age=0
User-Agent: Debian APT-HTTP/1.3 (1.2.35)


Ign:17 http://archive.ubuntu.com/ubuntu xenial/main amd64 Packages
4% [Working]GET /ubuntu/dists/xenial/main/binary-all/Packages.gz HTTP/1.1
Host: archive.ubuntu.com
Cache-Control: max-age=0
User-Agent: Debian APT-HTTP/1.3 (1.2.35)


Answer for: http://archive.ubuntu.com/ubuntu/dists/xenial/main/binary-all/Packages.gz
HTTP/1.1 404 Not Found
Date: Sat, 26 Aug 2023 20:59:46 GMT
Server: Apache/2.4.52 (Ubuntu)
Content-Length: 280
Connection: close
Content-Type: text/html; charset=iso-8859-1

Ign:18 http://archive.ubuntu.com/ubuntu xenial/main all Packages
4% [Working]GET /ubuntu/dists/xenial/restricted/binary-amd64/Packages.gz HTTP/1.1
Host: archive.ubuntu.com
Cache-Control: max-age=0
User-Agent: Debian APT-HTTP/1.3 (1.2.35)


Ign:9 http://security.ubuntu.com/ubuntu xenial-security/restricted amd64 Packages
4% [Working]GET /ubuntu/dists/xenial-security/restricted/binary-all/Packages.xz HTTP/1.1
Host: security.ubuntu.com
Cache-Control: max-age=0
User-Agent: Debian APT-HTTP/1.3 (1.2.35)


Answer for: http://security.ubuntu.com/ubuntu/dists/xenial-security/restricted/binary-all/Packages.xz
HTTP/1.1 404 Not Found
Date: Sat, 26 Aug 2023 21:01:44 GMT
Server: Apache/2.4.29 (Ubuntu)
Content-Length: 281
Connection: close
Content-Type: text/html; charset=iso-8859-1

Ign:11 http://security.ubuntu.com/ubuntu xenial-security/restricted all Packages
4% [Working]GET /ubuntu/dists/xenial-security/universe/binary-amd64/Packages.xz HTTP/1.1
Host: security.ubuntu.com
Cache-Control: max-age=0
User-Agent: Debian APT-HTTP/1.3 (1.2.35)


Ign:19 http://archive.ubuntu.com/ubuntu xenial/restricted amd64 Packages
4% [Working]GET /ubuntu/dists/xenial/restricted/binary-all/Packages.gz HTTP/1.1
Host: archive.ubuntu.com
Cache-Control: max-age=0
User-Agent: Debian APT-HTTP/1.3 (1.2.35)


Answer for: http://archive.ubuntu.com/ubuntu/dists/xenial/restricted/binary-all/Packages.gz
HTTP/1.1 404 Not Found
Date: Sat, 26 Aug 2023 21:01:47 GMT
Server: Apache/2.4.29 (Ubuntu)
Content-Length: 280
Connection: close
Content-Type: text/html; charset=iso-8859-1

Ign:20 http://archive.ubuntu.com/ubuntu xenial/restricted all Packages
4% [Working]GET /ubuntu/dists/xenial/universe/binary-amd64/Packages.gz HTTP/1.1
Host: archive.ubuntu.com
Cache-Control: max-age=0
User-Agent: Debian APT-HTTP/1.3 (1.2.35)


Ign:12 http://security.ubuntu.com/ubuntu xenial-security/universe amd64 Packages
4% [Working]GET /ubuntu/dists/xenial-security/universe/binary-all/Packages.xz HTTP/1.1
Host: security.ubuntu.com
Cache-Control: max-age=0
User-Agent: Debian APT-HTTP/1.3 (1.2.35)


Answer for: http://security.ubuntu.com/ubuntu/dists/xenial-security/universe/binary-all/Packages.xz
HTTP/1.1 404 Not Found
Date: Sat, 26 Aug 2023 21:03:45 GMT
Server: Apache/2.4.52 (Ubuntu)
Content-Length: 281
Connection: close
Content-Type: text/html; charset=iso-8859-1

Ign:13 http://security.ubuntu.com/ubuntu xenial-security/universe all Packages
4% [Working]GET /ubuntu/dists/xenial-security/multiverse/binary-amd64/Packages.xz HTTP/1.1
Host: security.ubuntu.com
Cache-Control: max-age=0
User-Agent: Debian APT-HTTP/1.3 (1.2.35)


Ign:21 http://archive.ubuntu.com/ubuntu xenial/universe amd64 Packages
4% [Working]GET /ubuntu/dists/xenial/universe/binary-all/Packages.gz HTTP/1.1
Host: archive.ubuntu.com
Cache-Control: max-age=0
User-Agent: Debian APT-HTTP/1.3 (1.2.35)


Answer for: http://archive.ubuntu.com/ubuntu/dists/xenial/universe/binary-all/Packages.gz
HTTP/1.1 404 Not Found
Date: Sat, 26 Aug 2023 21:03:47 GMT
Server: Apache/2.4.52 (Ubuntu)
Content-Length: 280
Connection: close
Content-Type: text/html; charset=iso-8859-1

Ign:22 http://archive.ubuntu.com/ubuntu xenial/universe all Packages
4% [Working]GET /ubuntu/dists/xenial/multiverse/binary-amd64/Packages.gz HTTP/1.1
Host: archive.ubuntu.com
Cache-Control: max-age=0
User-Agent: Debian APT-HTTP/1.3 (1.2.35)


Ign:23 http://archive.ubuntu.com/ubuntu xenial/multiverse amd64 Packages
Ign:15 http://security.ubuntu.com/ubuntu xenial-security/multiverse amd64 Packages
4% [Connecting to archive.ubuntu.com (185.125.190.36)] [Connecting to security.ubuntu.com (91.189.91.39)]GET /ubuntu/dists/xenial-security/multiverse/binary-all/Packages.xz HTTP/1.1
Host: security.ubuntu.com
Cache-Control: max-age=0
User-Agent: Debian APT-HTTP/1.3 (1.2.35)


GET /ubuntu/dists/xenial/multiverse/binary-all/Packages.gz HTTP/1.1
Host: archive.ubuntu.com
Cache-Control: max-age=0
User-Agent: Debian APT-HTTP/1.3 (1.2.35)


4% [Waiting for headers] [Waiting for headers]Answer for: http://security.ubuntu.com/ubuntu/dists/xenial-security/multiverse/binary-all/Packages.xz
HTTP/1.1 404 Not Found
Date: Sat, 26 Aug 2023 21:06:59 GMT
Server: Apache/2.4.29 (Ubuntu)
Content-Length: 281
Connection: close
Content-Type: text/html; charset=iso-8859-1

Ign:16 http://security.ubuntu.com/ubuntu xenial-security/multiverse all Packages
4% [Waiting for headers]GET /ubuntu/dists/xenial-security/main/binary-amd64/Packages.bz2 HTTP/1.1
Host: security.ubuntu.com
Cache-Control: max-age=0
User-Agent: Debian APT-HTTP/1.3 (1.2.35)


Answer for: http://archive.ubuntu.com/ubuntu/dists/xenial/multiverse/binary-all/Packages.gz
HTTP/1.1 404 Not Found
Date: Sat, 26 Aug 2023 21:06:59 GMT
Server: Apache/2.4.52 (Ubuntu)
Content-Length: 280
Connection: close
Content-Type: text/html; charset=iso-8859-1

Ign:24 http://archive.ubuntu.com/ubuntu xenial/multiverse all Packages
4% [Working]GET /ubuntu/dists/xenial-updates/main/binary-amd64/Packages.gz HTTP/1.1
Host: archive.ubuntu.com
Cache-Control: max-age=0
User-Agent: Debian APT-HTTP/1.3 (1.2.35)


4% [Working]^C
root@docker-desktop:/#
```

<details>
It seems to just progress really slow from the beginning (as you can see from the time of the log)

My docker for windows version is 4.17.0, Ill try to update to the latest version and see if it helps.

**Comment by Emilgardis** [comment]
alright, I think the best solution for you then is to just install cross from main

`cargo install cross --git https://github.com/cross-rs/cross`

**Comment by alloncm** [comment]
Thank you for your time!

BTW, any particular reason there isn't "cross 0.2.6" that at least updates the images used by default?

**Comment by Emilgardis** [comment]
Awesome! I'll close this as "just something strange happening on windows"

We have [some things](https://github.com/cross-rs/cross/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+milestone%3Av0.3.0) that should be fixed before 0.3.0

Can't/shouldn't release the bump in the base of the images as 0.2.6.

I'm planning to do a release soon(TM).
